content/en/monitors/guide/supress-alert-with-downtimes.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Suppress alerts with Downtimes
+kind: guide
+further_reading:
+- link: "api/v1/downtimes/"
+  tag: "Documentation"
+  text: "Downtime API reference"
+- link: "/monitors/downtimes/"
+  tag: "Documentation"
+  text: "Downtime documentation"
+---
+
+
+## Overview
+
+Use Downtimes to eliminate unnecessary alerts during scheduled maintenance, testing or auto scaling events.
+Use the [Downtime API][1] to manage advanced maintenance schedules format or to dynamically mute monitors, for example when resizing cloud instances.
+
+This guide describes how to configure downtimes for the following use cases:
+
+* [Downtime over the weekend](#downtime-over-the-week-end)
+* [Downtime outside of business hours](#downtime-outside-of-business-hours)
+* [Recurring downtime on nth weekday of the month](#recurring-downtime-on-nth-weekday-of-the-month)
+
+## Prerequisites
+
+Since this guide describes usage of the API, you will need an API key and an application key from an admin user. These are available in your [Datadog account API key page][2].
+Throughout this article, you will need to replace all occurrences of `<DATADOG_API_KEY>` and `<DATADOG_APP_KEY>` with your Datadog API key and your Datadog application key, respectively.
+
+This guide also assumes that you have a terminal with `CURL` and have reviewed the main [Downtime documentation page][5]
+
+**Note:** If you are in the Datadog EU site, use https://api.datadoghq.eu/api/  as the endpoint.
```

Comment:
```suggestion
```
So we should remove this and update all of the code examples to work with the site selector dropdown. The internal docs for how that works are here - https://datadoghq.atlassian.net/wiki/spaces/docs4docs/pages/1183023503/Regions

Let me know if you would like help with that.

Reply:
Help would definitely be welcome.